lib/renderers/curses/curses.c

```diff
@@ -221,7 +221,13 @@ render(const struct bm_menu *menu)
         doffset -= (prev ? prev : 1);
     }
 
-    draw_line(0, 0, "%*s%s", title_len, "", (menu->filter ? menu->filter + doffset : ""));
+    char *filter_text = (menu->filter ? menu->filter : "");
```

Comment:
should be `menu->filter ? menu->filter + doffset : "")` instead, adding + doffset later may cause invalid read.

Reply:
Also can be `const char *filter_text = ...` since not modified.